app_feup/lib/redux/action_creators.dart

```diff
@@ -65,19 +65,29 @@ ThunkAction<AppState> reLogin(username, password, faculty, {Completer action}) {
   };
 }
 
-ThunkAction<AppState> login(username, password, faculty, persistentSession,
+ThunkAction<AppState> login(username, password, faculties, persistentSession,
     usernameController, passwordController) {
   return (Store<AppState> store) async {
     try {
       store.dispatch(SetLoginStatusAction(RequestStatus.busy));
+
+      // TODO: multiple faculties support
```

Comment:
Can you leave a TODO pointing towards an issue that you create?

Reply:
I am goind to create an issue for that, however, it should bue discussed as I do not know exactly what the right approach should be in this situation